### DISCUSSION TITLE
Tell me about your pets or favorite animals!!!

### DISCUSSION THREAD

**Discussion by mueharac** [NAIST-SE/AttractingContributors#475] [Q&A]
Have you ever had any pets?
Please tell me how those are cute ♡

You have never had any pets?
Please tell me how your favorite animals are cute ♡

あなたのペットあるいは好きな動物について教えてください！

↓Repository↓
https://github.com/tabineko/favoriteanimals

**Comment by mueharac** [comment]
Still under seeking of collaborator. 
回答者募集中です！よろしくお願いします！

↓Repository↓
https://github.com/tabineko/favoriteanimals

**Comment by ani-hovhannisyan** [comment]
I added #6 animal. Please help me back.
discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git

**Comment by mueharac** [comment]
still open!
I can help you too.
